src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -1264,6 +1264,15 @@ namespace Akka.Actor
         public Akka.Actor.IActorRef Watch(Akka.Actor.IActorRef subject) { }
         public Akka.Actor.IActorRef WatchWith(Akka.Actor.IActorRef subject, object message) { }
     }
+    public sealed class IntentionalActorRestartException : Akka.Actor.AkkaException
```

Comment:
Added a new `IntentionalActorRestartException` and an `IntentionalRestart` auto-received message. I hope the naming makes it obvious why these exist.